### DISCUSSION TITLE
fmi2DoStep failed with status 3 (error).

### DISCUSSION THREAD

**Discussion by Yahiaassam** [OpenModelica/OpenModelica#9391] [Q&A]
Good morning all,

I would like to simulate an OM model in python using an FMU.
when I run the python program it shows this error "fmi2DoStep failed with status 3 (error)."

I don't see how to solve this problem

Any help will be welcome;

thank's.

**Comment by fhi2023** [comment]
I have the same issue

**Comment by casella** [comment]
Can you provide a test model to reproduce the issue?
